### ISSUE TITLE
Unable to copy with mouse when file contains new lines before resizing

### ISSUE THREAD

**Issue by andrewvaughanj** [wez/wezterm#1263]
### What Operating System(s) are you seeing this problem on?

Windows

### WezTerm version

wezterm 20211024-192045-f4ee31fe

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Using local build from `main` @ f4ee31fe

### Describe the bug

If I use `wezterm` (on Windows) to `ssh` to a Linux machine, and then I try to use the mouse to copy a file that has Linux line-endings (i.e., `\n`), then `wezterm`'s ability to block-select text with the mouse doesn't work.

**When writing up this issue, I realised that the issue "goes away" if you resize the `wezterm` window after `cat`ing the file**

If you have something like:

```
hello hello hello

new new new

lines lines lines
```

and you start copying from "lines lines lines" (from bottom to top), the selection with be dropped when you move to the new line between "lines lines lines" and "new new new", and if you continue dragging, `wezterm` will then select only the the first "n" of "new new new".

![image](https://user-images.githubusercontent.com/14277540/138671307-f8e4ead0-2755-4e0f-a10e-ed0e8e870b34.png)

### To Reproduce

- Start on Windows
- SSH to a Linux machine
- Run `echo -e "hello hello hello\n\nnew new new\n\nlines lines lines\n\n" > test.txt`
- Run `cat test.txt`
- **Do not resize the window**
- Try to copy text with the mouse

### Configuration

no config

### Expected Behavior

That dragging a selection across new lines will allow me to copy the whole selection (including new-line characters)

### Logs

Captured with `set WEZTERM_LOG=wezterm_gui::termwindow::mouse=trace,info`, but I think they're quite "uninteresting":

```
 2021-10-25T09:32:18.515Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (99, 432), screen_coords: (2362, 1235), mouse_buttons: NONE, modifiers: NONE }
 2021-10-25T09:32:18.619Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Press(Left), coords: (99, 432), screen_coords: (2362, 1235), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.651Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (98, 432), screen_coords: (2361, 1235), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.660Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (97, 432), screen_coords: (2360, 1235), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.667Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (96, 432), screen_coords: (2359, 1235), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.668Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (94, 432), screen_coords: (2357, 1235), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.681Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (88, 432), screen_coords: (2351, 1235), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.697Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (80, 429), screen_coords: (2343, 1232), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.698Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (79, 429), screen_coords: (2342, 1232), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.698Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (79, 428), screen_coords: (2342, 1231), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.713Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (73, 424), screen_coords: (2336, 1227), mouse_buttons: LEFT, modifiers: NONE }
 2021-10-25T09:32:18.730Z TRACE wezterm_gui::termwindow::mouseevent > MouseEvent { kind: Move, coords: (65, 420), screen_coords: (2328, 1223), mouse_buttons: LEFT, modifiers: NONE }
```

### Anything else?

It is interesting that this works if you resize the window after displaying it.

**Comment by davidrios** [comment]
I used to experience this (and it was extremely annoying), but it stopped after one of the latest releases, so I think this is already fixed. Could you confirm please?

**Comment by wez** [comment]
Likely the same thing as #1472 and resolved in 5608e9477b2b22a1bd8c5727440ae6a763c06b06
